src/main/java/org/jabref/gui/preferences/PreferencesDialogViewModel.java

```diff
@@ -174,6 +177,12 @@ public void storeAllSettings() {
             return;
         }
 
+        if (justImported) {
+            dialogService.notify(Localization.lang("Preferences were just imported. No additional changes to save."));
+            justImported = false;
```

Comment:
And this is not necessary :smile: , as no code will check it, because the function immediately returns after that.

Reply:
Oops, Git review moment.

By `this` in my comment I meant `justImported = false`